package.json

```diff
@@ -11,7 +11,8 @@
     "test:coverage": "lerna run test:coverage",
     "test:coverage:report": "lerna run test:coverage:report",
     "lint": "lerna run lint",
-    "lint:watch": "lerna run lint:watch"
+    "lint:watch": "lerna run lint:watch",
+    "postinstall": "lerna bootstrap"
```

Comment:
Do you have any prior art to cite for this trick from other monorepo projects?

Reply:
Does this mean we could remove the `lerna bootstrap` command from the `.travis.yml`?

Reply:
It's not on lerna.js.org but they have some notes on it in their FAQ. For package.json and even CI they are offering some tips.

https://github.com/lerna/lerna/blob/64b60a1831a6fc5f0d31ec8721f68fbc834e8caa/FAQ.md#root-packagejson

https://github.com/lerna/lerna/blob/5344820fc65da081d17f7fd2adb50ffe7101905b/commands/bootstrap/index.js#L66

While I was working on this large monorepo it tipped me off how some people use it.
https://github.com/ethereum/web3.js/blob/1.x/package.json#L23

Every npm script has a `pre` and `post` hook. Given this package.json we can easily demonstrate what is going on.

```json
{
  "name": "test-module",
  "private": true,
  "scripts": {
    "premockyeah": "echo premockyeah",
    "mockyeah": "echo mockyeah",
    "postmockyeah": "echo postmockyeah"
  }
}
```

`npm run mockyeah`

Output:

```
> test-module@ premockyeah
> echo premockyeah

premockyeah

> test-module@ mockyeah
> echo mockyeah

mockyeah

> test-module@ postmockyeah
> echo postmockyeah

postmockyeah
```

---

> Does this mean we could remove the lerna bootstrap command from the .travis.yml?

Yes

Reply:
Great info, thanks @tcrowe! Do you mind removing `lerna bootstrap` from the `.travis.yml` here then? I'm good to merge after that.